tests/foreman/ui/test_discoveredhost.py

```diff
@@ -100,53 +49,23 @@ def _is_host_reachable(host, retries=12, iteration_sleep=5, expect_reachable=Tru
         return bool(result.status)
 
 
-@pytest.mark.stubbed
-@pytest.mark.tier3
-def test_positive_pxe_based_discovery(session, provisioning_env):
-    """Discover a host via PXE boot by setting "proxy.type=proxy" in
-    PXE default
-
-    :id: 43a8857d-2f08-436e-97fb-ffec6a0c84dd
-
-    :Setup: Provisioning should be configured
-
-    :Steps: PXE boot a host/VM
-
-    :expectedresults: Host should be successfully discovered
-
-    :BZ: 1731112
-
-    :CaseImportance: Critical
-    """
-
-
-@pytest.mark.stubbed
-@pytest.mark.tier3
-def test_positive_pxe_less_with_dhcp_unattended(session, provisioning_env):
```

Comment:
we're not covering pxeless anywhere here now, so still are we good removing this stubbed test?

Reply:
We are covering it with auto-provision scenario

Reply:
https://github.com/SatelliteQE/robottelo/pull/12991/files#diff-922c0dec870899112b8342d4e6f4bac3e164afa097a4de130def5f7005bca3d5R157